### PR TITLE
removing excessive copying

### PR DESCRIPTION
**Pull request by nzardosh**

### CODE REVIEW

**Review by jgrosseo — DISMISSED** [review]

**Comment by davidrohr** [comment]
@TimoWilken @ktf : The FullCI is currently failing quite often since it is killed due to out of memory, could you check.

**Comment by jgrosseo** [comment]
@nzardosh As discussed, let's see first how we want the final table

**Comment by nzardosh** [comment]
Please do not merge. I intended to make a draft PR but didn't realise there was a pending PR on the branch

File: Analysis/Tasks/SkimmingTutorials/DataModel/LFDerived.h [hunk 1]
```diff
@@ -15,23 +15,41 @@
...
+
+DECLARE_SOA_TABLE(LFNucleiTracks, "AOD", "LFNUCLEITRACK", o2::soa::Index<>,
+                  lftrack::LFCollisionId,
+                  lftrack::Pt, lftrack::P,
+                  lftrack::Eta, lftrack::Phi,
+                  lftrack::TPCNSigma, lftrack::TOFNSigma);
+using LFNucleiTrack = LFNucleiTracks::iterator;
 } // namespace o2::aod
 
+enum PIDSpecies { El_PID = 0,
```
Comment (jgrosseo) [inline Analysis/Tasks/SkimmingTutorials/DataModel/LFDerived.h]: We shall discuss it is not better to have explicit names in the table and avoid the enum

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**